src/SDKs/Dns/Management.Dns/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure DNS Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure DNS Resources.")]
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
@karravi-msft you missed bumping up AssemblyVersion. Please update AssemblyVersion to 2.0.0.0